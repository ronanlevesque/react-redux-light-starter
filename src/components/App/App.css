.wrapper {
  display: flex;
  min-height: 100vh;
}

.content {
  margin: auto;
}

.title {
  text-align: center;
  color: #fff;
}

.logo {
  margin: auto;
  display: block;
  width: 120px;
  height: 120px;
}
