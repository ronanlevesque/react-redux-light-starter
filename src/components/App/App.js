import React from 'react';
import styles from './App.css';
import logo from '../../img/logo-react.png';

const App = () => (
  <div className={styles.wrapper}>
    <div className={styles.content}>
      <img className={styles.logo} src={logo} alt="React logo" />
      <h1 className={styles.title}>Hello world!</h1>
    </div>
  </div>
);

export default App;
