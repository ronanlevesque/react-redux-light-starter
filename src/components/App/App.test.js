import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';

test('AppTest', () => {

  const component = renderer.create(
    <App />
  );

  expect(component.toJSON()).toMatchSnapshot();
});
