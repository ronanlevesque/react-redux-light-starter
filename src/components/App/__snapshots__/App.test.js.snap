// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTest 1`] = `
<div
  className="wrapper"
>
  <div
    className="content"
  >
    <img
      alt="React logo"
      className="logo"
      src={undefined {}}
    />
    <h1
      className="title"
    >
      Hello world!
    </h1>
  </div>
</div>
`;
